VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionCommentsFragment.java

```diff
@@ -40,18 +42,15 @@
     private DiscussionComment discussionComment;
 
     @Inject
-    DiscussionAPI discussionAPI;
-
-    @Inject
-    DiscussionCommentsAdapter discussionCommentsAdapter;
-
-    @Inject
     Router router;
 
     @Inject
     Context context;
 
-    private static final Logger logger = new Logger(CourseDiscussionCommentsFragment.class.getName());
+    private DiscussionCommentsAdapter discussionCommentsAdapter;
+
+    @Nullable
```

Comment:
I don't think there is any point in annotating non-parameter variables as `Nullable` or `NonNull`, as they can either not be verified at all in static analysis (as here in the case of object/class variables), or their state can already be analyzed (e.g. method-local variable initializations). It's only useful for applying on parameters and return values, so that the analyzer knows what is expected there.